lib/ffmpeg/vaapi/transcoder.py

```diff
@@ -35,6 +35,9 @@ class TranscoderTest(BaseTranscoderTest):
         sw = (dict(maxres = (16384, 16384)), have_ffmpeg_decoder("vc1"), "vc1"),
         hw = (platform.get_caps("decode", "vc1"), have_ffmpeg_decoder("vc1"), "vc1"),
       ),
+      "av1" : dict(
+        hw = (platform.get_caps("decode", "av1_8"), have_ffmpeg_decoder("av1_qsv"), "av1_qsv"),
```

Comment:
"av1_qsv" does not belong in "vaapi" test subclass

Reply:
patch update